internal/generator/fluentd/sources.go

```diff
@@ -105,6 +105,7 @@ func ExcludeContainerPaths(namespace string) string {
 	for _, comp := range []string{constants.CollectorName, constants.ElasticsearchName, constants.KibanaName} {
```

Comment:
With LokiName="logging-loki" there's no need for special logic for loki, it is like any other component.
```suggestion
	for _, comp := range []string{constants.CollectorName, constants.ElasticsearchName, constants.KibanaName, constants.LokiName} {
```